server/src/services/boards/boards.go

```diff
@@ -133,7 +133,11 @@ func (s *BoardService) BoardOverview(_ context.Context, boardIDs []uuid.UUID, us
 }
 
 func (s *BoardService) Delete(_ context.Context, id uuid.UUID) error {
-	return s.database.DeleteBoard(id)
+	err := s.database.DeleteBoard(id)
+	if err != nil {
+		logger.Get().Errorw("unable to delete board", "err", err)
+	}
```

Comment:
Wouldn't it be better to log the error in the api layer?
Like this we would check twice if the error is not null

Reply:
In my opinion we should log the error as close to the source as possible.